content/blog/2020-10-tokio-0-3.md

```diff
@@ -0,0 +1,138 @@
+---
+date: "2020-10-14"
+title: "Announcing Tokio 0.3 and the path to 1.0"
+description: "October 14, 2020"
+---
+
+The Tokio team is excited to announce the release of Tokio 0.3, which is our beta release towards Tokio 1.0. The release fixes rough edges in the API discovered by customers using 0.2 and provides an opportunity for validating the changes before stabilizing them as part of 1.0. As most of these issues were small, we expect that upgrading from 0.2 to 0.3 will be easy.
+
+## Plan for 1.0
+
+The Tokio team plans to release Tokio 1.0 by the end of December 2020. This date is fast approaching. We ask that *you*, the Tokio community, try out Tokio 0.3 and give us feedback through [GitHub Issues](https://github.com/tokio-rs/tokio/issues) or our [Discord channel](https://discord.gg/tokio). Once we release Tokio 1.0, we will commit to the following stability guarantees:
+
+1. A minimum of 5 years of maintenance.
+2. A minimum of 3 years before a hypothetical 2.0 release.
+
+## What’s new?
+
+The main changes in Tokio 0.3 are:
+
+1. Changes to IO traits.
+2. New runtime builder.
+3. The I/O driver is overhauled
+4. The API is future proofed.
+
+You can find the full list on the [changelog](TODO LINK).
```

Comment:
This will be the github release: https://github.com/tokio-rs/tokio/releases/tag/tokio-0.3.0